javascript-forms/form-values.js

```diff
@@ -0,0 +1,12 @@
+const $contactForm = document.querySelector('#contact-form');
+function handleSubmit(event) {
+  const messageData = {};
+  messageData.name = $contactForm.elements.name.value;
+  messageData.email = $contactForm.elements.email.value;
+  messageData.message = $contactForm.elements.message.value;
+  console.log('messageData:', messageData);
+  event.target.reset();
+  event.preventDefault();
```

Comment:
This should be the first thing in the function, because it's the first action we want to take place